src/crashlytics/buildToolsJarHelper.ts

```diff
@@ -0,0 +1,69 @@
+import * as fs from "fs-extra";
+import * as os from "os";
+import * as path from "path";
+import * as spawn from "cross-spawn";
+
+import * as downloadUtils from "../downloadUtils";
+import { FirebaseError } from "../error";
+import { logger } from "../logger";
+import * as rimraf from "rimraf";
+import * as utils from "../utils";
+
+const JAR_CACHE_DIR =
+  process.env.FIREBASE_CRASHLYTICS_BUILDTOOLS_PATH ||
+  path.join(os.homedir(), ".cache", "firebase", "crashlytics", "buildtools");
+
+const JAR_VERSION = "2.9.1";
+const JAR_URL = `https://dl.google.com/android/maven2/com/google/firebase/firebase-crashlytics-buildtools/${JAR_VERSION}/firebase-crashlytics-buildtools-${JAR_VERSION}.jar`;
+
+/**
+ * Returns the path to the jar file, downloading it if necessary.
+ */
+export async function fetchBuildtoolsJar(): Promise<string> {
+  // If you set LOCAL_JAR to a path it will override the downloaded buildtools.jar
+  if (process.env.LOCAL_JAR) {
```

Comment:
`LOCAL_JAR` is a pretty generic variable. Would `CRASHLYTICS_LOCAL_JAR` be better/safer, if more verbose?